bin/elasticsearch.bat

```diff
@@ -33,6 +33,9 @@ FOR /F "usebackq tokens=1* delims= " %%A IN (!params!) DO (
 	)
 )
 
+set HOSTNAME=
+for /f "delims=" %%a in ('hostname') do @set HOSTNAME=%%a
```

Comment:
Argh, didn't see the uppercase, fixed.
Well, I wasn't successful running elasticsearch on Windows 2000 ("Unsupported major.minor version 51.0") but I can confirm that the solution for setting the hostname works:
![2015-02-06_2117](https://cloud.githubusercontent.com/assets/1188538/6087267/bcba0a8c-ae49-11e4-8e01-aa41fe4a9601.png)